File: exams/frmPrintFrom.vb
Public Class frmPrintFrom

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        If txtFrom.Text.Trim <> Nothing And txtTo.Text.Trim <> Nothing Then
            cont = True
            row_from = txtFrom.Text
            row_to = txtTo.Text
            Me.Close()
        End If
    End Sub
    Private Function isvalid() As Boolean

    End Function

    Private Sub frmPrintFrom_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load

    End Sub
End Class

File: exams/frmPrintFrom.cs
using System;
using Microsoft.VisualBasic.CompilerServices;

namespace exams
{
    public partial class frmPrintFrom
    {
        public frmPrintFrom()
        {
            InitializeComponent();
            _Button1.Name = "Button1";
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            if (!string.IsNullOrEmpty(txtFrom.Text.Trim()) & !string.IsNullOrEmpty(txtTo.Text.Trim()))
            {
                publicSubsNFunctions.cont = true;
                publicSubsNFunctions.row_from = Conversions.ToInteger(txtFrom.Text);
                publicSubsNFunctions.row_to = Conversions.ToInteger(txtTo.Text);
                Close();
            }
        }

        private bool isvalid()
        {
            return default;
        }

        private void frmPrintFrom_Load(object sender, EventArgs e)
        {
        }
    }
}